### PR TITLE
Fix way we force es secret reconcile

### PR DESCRIPTION
**Pull request by kevinearls**
Signed-off-by: Kevin Earls <kearls@redhat.com>

### CODE REVIEW

**Comment by kevinearls** [comment]
@jpkrohling Please review

File: scripts/cert_generation.sh [hunk 1]
```diff
@@ -209,7 +209,9 @@ function generate_certs() {
 
   # For TRACING-1631 - if we can't find the namespace in the cert it's bad, regenerate everything
   if [ $REGENERATE_NEEDED = 0 ] && [ "${component}" == "elasticsearch" ] && [ -f ${WORKING_DIR}/logging-es.crt ]  ; then
-    openssl x509 -in ${WORKING_DIR}/logging-es.crt -text | grep -q "DNS:elasticsearch.${NAMESPACE}.svc"
+    # Make sure the SAN contains both "DNS:elasticsearch.${NAMESPACE}.svc.cluster.local" AND "DNS:elasticsearch.${NAMESPACE}.svc.
+    # The latter may be followed by a comma or whitespace
```
Comment (jpkrohling) [inline scripts/cert_generation.sh:213]: Sounds strange. Any reasons why?

**Review by jpkrohling — COMMENTED** [review]
Looks good, just a "why" before merging :-)

File: scripts/cert_generation.sh (hunk 1 above)
Comment (kevinearls) [reply scripts/cert_generation.sh:213]: Sorry, let me fix that

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**